<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolButton" name="userButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>用户</string>
    </property>
   </widget>
   <widget class="QToolButton" name="queryButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>查询</string>
    </property>
   </widget>
   <widget class="QToolButton" name="learnButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>学习</string>
    </property>
   </widget>
   <widget class="QToolButton" name="reviewButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>复习</string>
    </property>
   </widget>
   <widget class="QToolButton" name="achievementButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>成就</string>
    </property>
   </widget>
   <widget class="QToolButton" name="gameButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>游戏</string>
    </property>
   </widget>
   <widget class="QToolButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>600</width>
      <height>500</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>userButton</tabstop>
  <tabstop>queryButton</tabstop>
  <tabstop>learnButton</tabstop>
  <tabstop>reviewButton</tabstop>
  <tabstop>achievementButton</tabstop>
  <tabstop>gameButton</tabstop>
  <tabstop>exitButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
